### ISSUE TITLE
Better message for everyone

### ISSUE THREAD

**Issue by apple502j** [LLK/scratch-www#3635]
[A lot of Japanese Scratchers are now reporting a "bug"](https://scratch.mit.edu/discuss/post/3805257/), saying a text with red or orange background is shown. Of course this is a message about the Mandrill outage, but for those who don't read English, it's some kind of error message shown at the top of all pages.

[Others on QaS forum](https://scratch.mit.edu/discuss/post/3805249/) thought the account was hacked and something was changed.

Basically, messages without translation is bad - especially those which will be seen by many. If it's impossible to let them translate and review before, maybe at least consider using Google Translate?

What do you think?

**Comment by benjiwheeler** [comment]
Unfortunately, we didn't have time for the string to go through the normal translation process. This planned email outage by Mandrill was unusual and difficult to work around, but ideally we'd have the strings in place earlier so translators can translate them.

**Comment by apple502j** [comment]
Also, this message is shown *everywhere* - why not just use News on front page and/or Announcements?

**Comment by chrisgarrity** [comment]
I do think this was a pretty unusual case. Generally we don't consider Google translation as new strings are going to be used for a long time and we want them to make sense to everyone. When there's a new page or content changes, it's usually fine for translations to catch up over the following weeks and months.

In this case the message was a banner on all pages, and it was of limited time duration. It may be that using Google translate ourselves to generate a message would have been better than no translation.

**Comment by benjiwheeler** [comment]
Well said.

I would like to understand the comments on the Japanese forum post, to
better understand the ways that this was confusing and alarming to users.

**Comment by BryceLTaylor** [comment]
I'm going to close this issue, since the banner has been down for quite some time now.
